pkg/cmd/deploy.go

```diff
@@ -21,10 +21,9 @@ import (
 	"io"
 	"os/exec"
 
+	"github.com/ghodss/yaml"
```

Comment:
Other being 3rd party libraries? If so, then +1. Have followed that in previous projects. Totally makes sense. This policy, if agreed upon, should go into the development doc.